src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Dto/RemoteVettingProcessDto.php

```diff
@@ -70,21 +78,28 @@ public static function deserialize($serialized)
      */
     public static function updateState(RemoteVettingProcessDto $process, RemoteVettingState $state)
     {
-        return new self($process->getProcessId(), $process->getToken(), $state, $process->getAttributes());
+        return new self($process->getProcessId(), $process->getToken(), $state, $process->getAttributes(), $process->getIdentityProviderName());
     }
 
     /**
      * @param ProcessId $processId
      * @param RemoteVettingTokenDto $token
      * @param RemoteVettingState $state
      * @param AttributeListDto $attributes
+     * @param string $identityProviderName
      */
-    private function __construct(ProcessId $processId, RemoteVettingTokenDto $token, RemoteVettingState $state, AttributeListDto $attributes)
-    {
+    private function __construct(
+        ProcessId $processId,
+        RemoteVettingTokenDto $token,
+        RemoteVettingState $state,
+        AttributeListDto $attributes,
+        $identityProviderName
+    ) {
         $this->processId = $processId;
         $this->token = $token;
         $this->state = $state;
         $this->attributes = $attributes;
+        $this->identityProviderName = $identityProviderName;
```

Comment:
Please add input guarding for this new field.

Reply:
Done!

Reply:
I don't see it yet? Forgot to push this?

Reply:
No, I did added it to the static factory method and not the constructor. Moved it.